src/App.js

```diff
@@ -13,7 +13,13 @@ export default function App() {
 
   function createCards(characters) {
     const cards = characters.map(character =>
```

Comment:
You could already use de-structuring assignment here!
```suggestion
    const cards = characters.map(({image, house, name, species, gender}) =>
```